app/views/mailers/notify_mailer/tag_moderator_confirmation_email.html.erb

```diff
@@ -24,23 +24,25 @@
   <a href="<%= app_url(tag_moderation_url) %>">tag moderation</a> page.</strong>
 </p>
 
-<p>
-  To aid in communication, we’ve added you to the following <%= community_name %> Connect channels:
-</p>
-<ul>
-  <li>
-     <a href="<%= app_url("/connect/tag-moderators") %>">Tag Moderators</a>
-  </li>
-  <% if @channel_slug %>
+<% if FeatureFlag.enabled?(:connect) %>
+  <p>
+    To aid in communication, we’ve added you to the following <%= community_name %> Connect channels:
+  </p>
+  <ul>
     <li>
-      <a href="<%= app_url("/connect/#{@channel_slug}") %>">#<%= @tag.name %></a>
+      <a href="<%= app_url("/connect/tag-moderators") %>">Tag Moderators</a>
     </li>
-  <% end %>
-</ul>
+    <% if @channel_slug %>
+      <li>
+        <a href="<%= app_url("/connect/#{@channel_slug}") %>">#<%= @tag.name %></a>
```

Comment:
**(comment, non-blocking):** I thought we could use a path helper here and for tag_moderators above, but weirdly enough connect_path doesn't behave as a I thought it would.

Rails.application.routes.url_helpers.connect_path("something")
=> "/locale/something/connect"

Reply:
Normally I'd get curious and investigate but given that we're about to remove all of this I'll just let it go.